Gemfile.lock

```diff
@@ -140,14 +287,15 @@ DEPENDENCIES
   rake
   rest-client
   rspec
+  rubocop
   sass
   simplecov
   sinatra
   sinatra-activerecord
   slim
 
 RUBY VERSION
-   ruby 2.3.0p0
+   ruby 2.4.0p0
 
 BUNDLED WITH
-   1.13.6
+   1.14.3
```

Comment:
This is the part that may potentially break. The Heroku docs specify that every Ruby project is built with Bundler 1.13.7. 1.13.6 refused to install dependencies because of an issue with `pry`.